cmd/lakefs/cmd/root.go

```diff
@@ -96,7 +96,7 @@ func newConfig() (config.Config, error) {
 	if name == config.QuickstartConfiguration {
 		validateQuickstartEnv(cfg.GetBaseConfig())
 	}
-	return cfg.GetBaseConfig(), nil
+	return cfg, nil
```

Comment:
Not sure where to put it -
But I think that the `catalog.New()` in `run.go` should also be called with `cfg` (instead of `baseCfg`).